### PR TITLE
5322: Edit description

### PR DESCRIPTION
**Pull request by talawson05**
Tests currently ignored due to remaining dev work
Propose to merge as is, to work on next story, and keep 6809 in active. Will then raise second PR removing the ignore tags once dev complete

[AB#6809](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/6809)
[AB#5322](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/5322)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Objects/Pages/OrderForm.cs [hunk 1]
```diff
@@ -8,7 +8,13 @@ public sealed class OrderForm
 		public By DeleteOrderButton => CustomBy.DataTestId("delete-order-button");
 		public By PreviewOrderButton => CustomBy.DataTestId("preview-order-button");
 		public By SubmitOrderButton => CustomBy.DataTestId("submit-order-button");
+		public By PageTitle => By.CssSelector("[data-test-id$='-page-title']");
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Objects/Pages/OrderForm.cs:11]: does this need the `$`?

Comment (talawson05) [reply src/OrderFormAcceptanceTests.Objects/Pages/OrderForm.cs:11]: yes that means string ends with

File: src/OrderFormAcceptanceTests.TestData/Order.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                            LastUpdatedBy=@lastUpdatedBy
+                        WHERE OrderId=@orderId";
+            SqlExecutor.Execute<Order>(connectionString, query, this);
+        }
+
+        public void Delete(string connectionString)
+        {
+            var query = @"DELETE FROM Order WHERE OrderId=@orderId";
+            SqlExecutor.Execute<Order>(connectionString, query, this);
+        }
```
[the remark this answers is not in the document]
Comment (talawson05) [reply src/OrderFormAcceptanceTests.TestData/Order.cs:68]: Retrieve yes, Create and delete not here but manually in SQL studio

File: src/OrderFormAcceptanceTests.Objects/Pages/OrderForm.cs (hunk 1 above)
Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Objects/Pages/OrderForm.cs:11]: All good then :)

File: src/OrderFormAcceptanceTests.TestData/Order.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.TestData/Order.cs:68]: Happy with that

**Pull request merged**